node/src/accountant/scanners_tools.rs

```diff
@@ -531,9 +518,52 @@ mod tests {
             response_skeleton_opt: None,
         };
 
-        let (ok, err) = separate_early_errors(&sent_payable, &Logger::new("test"));
+        let (ok, err) = separate_early_errors(&sent_payable, &Logger::new(test_name));
 
         assert_eq!(ok, vec![payable_ok]);
-        assert_eq!(err, vec![error])
+        assert_eq!(err, vec![error.clone()]);
+        TestLogHandler::new().exists_log_containing(&format!(
+            "WARN: {}: Outbound transaction failure due to '{:?}",
+            test_name, error
+        ));
+    }
+
+    #[test]
+    #[should_panic(expected = "Unable to calculate elapsed time for the scan.")]
+    fn update_timestamp_and_log_panics_if_timestamp_is_wrong() {
```

Comment:
I think we should keep it. In the end, it's just another safety measure.

Reply:
Hmm...then could say there plenty of untested places  because there are places like these, even after the same fn call, which would then deserve it too. 
But I don't think it deserves it. For me it is a spring of confusement because one's must ask why this case is different from others :man_shrugging: 

I myself wouldn't hesitate and would delete it straight away. What shall I do with you?